src/userStatusOverview/types.ts

```diff
@@ -1,26 +1,36 @@
+export enum UserStatusOverviewStatus {
+  Offline,
```

Comment:
Add to all fields a value, please

Reply:
this kind of declaration is allowed by typescript. typescript do auto increment of value itself

Reply:
I know that is allowed but I think that we should not rely on the order here.